blazorbootstrap/Components/Toasts/Toasts.razor.cs

```diff
@@ -26,7 +26,7 @@ protected override void OnInitialized()
         base.OnInitialized();
     }
 
-    private void OnNotify(ToastMessage toastMessage)
+    private async void OnNotify(ToastMessage toastMessage)
```

Comment:
Async void should be changed to 

```suggestion
    private async Task OnNotify(ToastMessage toastMessage)
```

Reply:
To apply this I'll have to change the event this method is added to to `Func<ToastMessage, Task>` but the tasks would be still thown away and wouldn't be awaited except when `ToastService.Notify` would be async which is a public API change.
Isn't it exceptable to throw the task already there away?

Reply:
https://learn.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void

Reply:
I know that async void should be avoided but it wouldn't realy make a difference. But anyway I'll change it.

Reply:
We can have two events, one sync and another async

Reply:
How exactly do you mean that? So two events, toasts component subscribes to both and what should invoke the other event? Maybe a NotifyAsync method in the ToastService?

Reply:
Yes, notifyAsync in ToastService, and move this statechanged to ToastService

Reply:
I would add this method
```csharp
Delegate[] subscribers = OnNotify?.GetInvocationList() ?? Array.Empty<Delegate>();
if (subscribers.Length > 0)
{
    await Task.WhenAll(subscribers.Select(d =>
    {
        Func<ToastMessage, Task> subscriber = (Func<ToastMessage, Task>)d;
        return subscriber.Invoke(toastMessage);
    }));
}
```
but two seperate events wouldn't be usefull because then there also have to be two seperate OnNotify in the toasts component. But we can still use the same event for the sync method right?.

> move this statechanged to ToastService

This isn't possible. StateHasChanged is a method of ComponentBase so it have to be called in the component.

Are you fine with the suggested method or is there something else?

Reply:
This seems good